src/components/DonorDashboard/DonorDashboard.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { Grid, GridItem, Box, Text } from '@chakra-ui/react';
-import { getDonationStatus } from '../../utils/donorUtils';
+import { getDonationStatus } from '../../utils/DonorUtils';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../../utils/DonorUtils"